### ISSUE TITLE
PSCIS submission

### ISSUE THREAD

**Issue by NewGraphEnvironment** [NewGraphEnvironment/fish_passage_fraser_2023_reporting#3]
The data will need to get from the QGIS/mergin project fieldform to the PSCIS submission template.

The process to do that is next on the list to get done so please stand by.  In the meantime though, please be aware that many of the sites that will pop with lots of habitat upstream were likely already assessed. We can see that in Q and have a look at the Peace QGIS project to see how we organize the layers in groups/directories and present in the `Reporting` theme.

In a perfect world we for sure submit that data to PSCIS first so the `bcfishpass.crossings_vw` layer will tie them together but for this use case that will need to wait.  

In the mean time we can make issues for individual sites here in the repo (similiar to the format done for Peace 2023) to keep track of site detail for already assessed sites that is not already in the field form comments and needs to be added once this script is ready (or turned into one big function in fpr - https://github.com/NewGraphEnvironment/fish_passage_peace_2023_reporting/blob/main/scripts/pscis_export.R )

**Comment by lucy-schick** [comment]
2023 phase 1 data had been successfully submitted. See issue https://github.com/NewGraphEnvironment/fish_passage_fraser_2023_reporting/issues/108.

Closing in favour of https://github.com/NewGraphEnvironment/fish_passage_guidebook/issues/13
